media/js/base/mozilla-monitor-button.js

```diff
@@ -20,17 +20,41 @@ Mozilla.MonitorButton = (function() {
     }
 
     var buttonURL = monitorButton.getAttribute('href');
+    // strip url to everything after `?`
+    buttonURL = buttonURL.match(/\?(.*)/)[1];
+
     var destURL = monitorButton.getAttribute('data-action') + 'metrics-flow';
 
-    fetch(destURL).then(function(resp) {
+    // collect values from monitor button
+    var utmParams = window._SearchParams.queryStringToObject(buttonURL);
+
+    console.log(utmParams);
+
+    var utmSource = utmParams.utm_source;
```

Comment:
Nit: these values only seem to be used once, so are they worth storing as variables? Could this be simplified to:

```javascript
destURL += '&' + params.utm_source;
```